pkg/cmd/variables/create/create.go

```diff
@@ -0,0 +1,86 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+
+	msg "github.com/aziontech/azion-cli/messages/variables"
+	api "github.com/aziontech/azion-cli/pkg/api/variables"
+
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/spf13/cobra"
+)
+
+type Fields struct {
+	Key    string
+	Value  string
+	Secret bool
+	Path   string
+}
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	fields := &Fields{}
+
+	cmd := &cobra.Command{
+		Use:           msg.CreateUsage,
+		Short:         msg.CreateShortDescription,
+		Long:          msg.CreateLongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        $ azioncli variables create --key "key" --value "value" --secret false
```

Comment:
I think we can put an actual good example, that theoretically could be copy+pasted, such as:
`azioncli variables create --key "Content-Type" --value "string" --secret false`